dashboard/app/models/pd/international_opt_in.rb

```diff
@@ -68,7 +75,7 @@ def validate_required_fields
   def self.options
     entry_keys = {
       gender: %w(male female non_binary not_listed none),
-      schoolCountry: %w(canada chile israel malaysia mexico thailand),
+      schoolCountry: %w(canada chile colombia israel malaysia mexico thailand),
```

Comment:
what is the significance of this list of countries?

Reply:
no idea :smile: 

I was just told we want to add Colombia to the list of countries in the dropdown, and add these custom options for school data for when it's selected.